static/app-strings.json

```diff
@@ -532,7 +536,7 @@
   "Not enough credits": "Not enough credits",
   "You have %credit_amount% in unclaimed rewards.": "You have %credit_amount% in unclaimed rewards.",
   "In response to a complaint we received under the US Digital Millennium Copyright Act, we have blocked access to this channel from our applications.": "In response to a complaint we received under the US Digital Millennium Copyright Act, we have blocked access to this channel from our applications.",
-  "Read More": "Read More",
```

Comment:
Please put this back

Reply:
Well, as of today, there is no string with this format in the source code. It was used only once in "\ui\component\channelContent\channleContent.jsx". The only one is "Read more." in all the files. And in the file with strings for translations there is in addition the form without dot at the end inserted by infinite-persistence for future needed cases. I will put back anyway in case there will some button.